src/components/ui/Dialog.tsx

```diff
@@ -63,20 +63,30 @@ const DialogContent = forwardRef<
       {...props}
     >
       {children}
-      <DialogClose className={cn('absolute right-3 top-3')}>
+      {/* Move DialogClose to the header */}
+
+      {/* <DialogClose className={cn('')}>
         <Icon icon={CrossIcon} color="black" size={3.5} />
         <span className="sr-only">Close</span>
-      </DialogClose>
+      </DialogClose> */}
     </PrimitiveContent>
   </DialogPortal>
 ));
 DialogContent.displayName = PrimitiveContent.displayName;
 
 const DialogHeader = ({ className, ...props }: HTMLAttributes<HTMLDivElement>) => (
-  <div
-    className={cn('h-12 px-4 border-b border-gray-light', 'flex items-center', className)}
+  // Add a new parent div
+  <div className={cn('h-12 px-4 border-b border-gray-light', 'flex items-center',)}>
+    <div
+    className={cn(className)}
     {...props}
   />
+    <DialogClose className={cn('absolute right-3')}>
+        <Icon icon={CrossIcon} color="black" size={3.5} />
+        <span className="sr-only">Close</span>
+      </DialogClose>
+  </div>
+  
```

Comment:
> Your instructions said that "The component structure and styles of DrawerHeader in src/components/ui/Drawer.tsx should be similar," however, I didn't need to make any changes there.

I apologize for the confusion. What I meant was that the `DialogHeader` should be the same as the `DrawerHeader`, apart from its name. Thus, basically, the only you need to do is to copy and paste from `DrawerHeader` to `DialogHeader.`

Reply:
Sorry, my bad. It's my first contribution. Just to clarify, I need to undo the changes I made and do this now instead, right?

Reply:
No problems! Please practice in our project. We, too, are quite new to accepting contributors.

We don't mind how you commit changes as long as the final code is ok!